cmpserver/plugin/plugin.go

```diff
@@ -106,14 +105,35 @@ func runCommand(ctx context.Context, command Command, path string, env []string)
 	logCtx.WithFields(log.Fields{"duration": duration}).Debug(output)
 
 	if err != nil {
-		err := newCmdError(args, errors.New(err.Error()), strings.TrimSpace(stderr.String()))
+		err := newCmdError(argsToLog, errors.New(err.Error()), strings.TrimSpace(stderr.String()))
 		logCtx.Error(err.Error())
 		return strings.TrimSuffix(output, "\n"), err
 	}
 
 	return strings.TrimSuffix(output, "\n"), nil
 }
 
+// getCommandArgsToLog represents the given command in a way that we can copy-and-paste into a terminal
+func getCommandArgsToLog(cmd *exec.Cmd) string {
+	var argsToLog []string
+	for _, arg := range cmd.Args {
+		if strings.ContainsRune(arg, ' ') {
+			// json-encode to add quotes and escape any internal quotes
+			encodedArg, err := json.Marshal(arg)
```

Comment:
Would it maybe more convenient to use `strconv.Quote` here instead of using JSON marshaling?

Reply:
Good idea, will do